edgeless_node/src/resources/dda/mod.rs

```diff
@@ -90,7 +91,8 @@ impl DDAResource {
         let dda_pub_array = match dcp {
             Ok(dda_array) => dda_array,
             Err(err) => {
-                eprintln!("Error parsing input dda_com_publication_mapping JSON: {}", err);
+                log::error!("Error parsing input dda_com_publication_mapping JSON: {}", err);
+                //eprintln!("Error parsing input dda_com_publication_mapping JSON: {}", err);
```

Comment:
The same comment above applies here to `process::exit()` and also to the `panic!()` below.

As a general rule, I would use `panic!()` only for when invariants are broken or assertions fail, i.e., for _bugs_, while in this case the panic would be triggered by a connection error, which is a run-time error.

Reply:
Same as above

Reply:
will go for an update on the exit / panic part as discussed in Chanaia in a separate pull request